evadb/third_party/databases/sqlite/sqlite_handler.py

```diff
@@ -146,3 +151,45 @@ def execute_native_query(self, query_string: str) -> DBHandlerResponse:
             return DBHandlerResponse(data=self._fetch_results_as_df(cursor))
         except sqlite3.Error as e:
             return DBHandlerResponse(data=None, error=str(e))
+
+    def _sqlite_to_python_types(self, sqlite_type: str):
```

Comment:
How about other handlers? Are they going to inherit this one?

Reply:
Nope, each database has its nuances and has to implement its own.